basepage/templates/basepage/base.html

```diff
@@ -43,6 +43,7 @@ <h1>
 
     {%else%}
     <a href="{% url 'login_user' %}">login</a>
+    <a href="{% url 'signup' %}">register</a>
```

Comment:
On the other PR I said something about using relative path. I meant this. This is the django way of referencing internal paths. These tags `{% something %}` are called template tags. The url template tag solves url referenced by their path names using a reverse. You can even configure your own template tags. You can give it a read on the documentation if you are curious.